package.json

```diff
@@ -5,8 +5,8 @@
   "main": "index.js",
   "scripts": {
     "setup-dbs": "psql -f ./db/setup.sql",
-    "seed": "node ./db/seeds/run-seed.js",
-    "test": "jest",
+    "seed": "PGDATABASE=nc_news node ./db/seeds/run-seed.js",
```

Comment:
we should not be setting these variables here as they will be publicly viewable. We should be setting these variables in .env files. Please remove the setting of PGDATABASE from your scripts and set it using dotenv.